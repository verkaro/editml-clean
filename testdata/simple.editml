Welcome to the {+full+} test suite.
This line tests a simple deletion{- of this extra text-}.
This one tests a {=highlight=}. {>This is a comment and should be removed.<}

Let's test structural edits. We will move C to the end of the line.
A. B. {move~C.~part1}
And now we place it here: {move:part1}

Finally, let's test a copy.
Original: {copy~Source~part2}. Copied: {copy:part2}.
