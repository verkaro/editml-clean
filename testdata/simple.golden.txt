Welcome to the full test suite.
This line tests a simple deletion.
This one tests a highlight. 

Let's test structural edits. We will move C to the end of the line.
A. B. 
And now we place it here: C.

Finally, let's test a copy.
Original: Source. Copied: Source.
